server/src/main/java/org/spine3/server/projection/ProjectionRepository.java

```diff
@@ -337,20 +338,42 @@ void store(Collection<P> projections) {
      * Updates projections from the event stream obtained from {@code EventStore}.
      */
     public void catchUp() {
```

Comment:
While the implementation looks logical enough, it will probably take ages to complete in an AppEngine application.

Let's discuss the example I have. It's easier to talk about it, than describe in a written form.